### PR TITLE
refined the topology-specific config procedure

### PR DESCRIPTION
**Pull request by klgill**
@matbu Can you please review my edits to the "Retrieving topology-specific service configuration" procedure and address the questions I've asked on this PR? 
Thank you.

### CODE REVIEW

File: docs_user/modules/proc_retrieving-topology-specific-service-configuration.adoc [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 = Retrieving topology-specific service configuration
 
+Before you migrate your databases to the {rhos_long} control plane, retrieve the topology-specific service configuration from your {rhos_prev_long} ({OpenStackShort}) environment. You need this configuration for the following reasons:
+
+* To check your current database for inaccuracies
+* To ensure that you have the data you need before the migration
+* To compare your {OpenStackShort} database with the adopted {rhos_acro} database
```
Comment (klgill) [inline docs_user/modules/proc_retrieving-topology-specific-service-configuration.adoc:9]: @matbu Can you confirm that the introduction is accurate?

Comment (matbu) [reply docs_user/modules/proc_retrieving-topology-specific-service-configuration.adoc:9]: Yes +1

File: docs_user/modules/proc_retrieving-topology-specific-service-configuration.adoc [hunk 2]
```diff
@@ -60,21 +65,21 @@ export PULL_OPENSTACK_CONFIGURATION_NOVA_COMPUTE_HOSTNAMES=$(oc run mariadb-clie
...
 export PULL_OPENSTACK_CONFIGURATION_NOVAMANAGE_CELL_MAPPINGS=$($CONTROLLER1_SSH sudo podman exec -it nova_api nova-manage cell_v2 list_cells)
 echo "$PULL_OPENSTACK_CONFIGURATION_NOVAMANAGE_CELL_MAPPINGS"
 ----
 +
-After the source control plane services shutdown, if either of the exported
-values lost, it could be no longer evaluated again. Preserving the exported
-values in an environment file should protect you from such a situation.
+[IMPORTANT]
+After the source control plane services are shut down, if any of the exported
+values are lost, the values cannot be evaluated again. To avoid data loss, preserve the exported values in an environment file before shutting down the control plane services.
```
[the remark this answers is not in the document]
Comment (matbu) [reply docs_user/modules/proc_retrieving-topology-specific-service-configuration.adoc:77]: @klgill I think @bogdando is the autor of this part, maybe @bogdando can confirm this updates ?

[the remark this answers is not in the document]
Comment (bogdando) [reply docs_user/modules/proc_retrieving-topology-specific-service-configuration.adoc:77]: the values cannot be evaluated again - if we re-run the commands, they will fail as there are no longer control plane running on the source cloud to get the data from (and those cannot be started back).

File: docs_user/assemblies/assembly_migrating-databases-to-the-control-plane.adoc [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 = Migrating databases to the control plane
 
-To begin creating the control plane, enable backend services and import the databases from your original {rhos_prev_long} {rhos_prev_ver} deployment.
+To begin creating the control plane, enable back-end services and import the databases from your original {rhos_prev_long} {rhos_prev_ver} deployment.
```
Comment (JoanneOFlynn2018) [inline docs_user/assemblies/assembly_migrating-databases-to-the-control-plane.adoc:7]: Is this assembly/user story just the beginning (To begin creating the control plane) of the creation of the control plane? I'd like to see a complete picture (super procedure) somewhere of the list of user stories in order, that you must complete.

File: docs_user/modules/proc_retrieving-topology-specific-service-configuration.adoc [hunk 4]
```diff
@@ -2,6 +2,12 @@
...
 
+Before you migrate your databases to the {rhos_long} control plane, retrieve the topology-specific service configuration from your {rhos_prev_long} ({OpenStackShort}) environment. You need this configuration for the following reasons:
+
+* To check your current database for inaccuracies
+* To ensure that you have the data you need before the migration
+* To compare your {OpenStackShort} database with the adopted {rhos_acro} database
+
 .Prerequisites
 
 * Define the following shell variables. The values that are used are examples. Replace these example values with values that are correct for your environment:
```
Comment (JoanneOFlynn2018) [inline docs_user/modules/proc_retrieving-topology-specific-service-configuration.adoc:13]:
```suggestion
* Define the following shell variables. Replace the example values with values that are correct for your environment:
```
Removed passive

File: docs_user/modules/proc_retrieving-topology-specific-service-configuration.adoc [hunk 5]
```diff
@@ -60,21 +65,21 @@ export PULL_OPENSTACK_CONFIGURATION_NOVA_COMPUTE_HOSTNAMES=$(oc run mariadb-clie
...
 export PULL_OPENSTACK_CONFIGURATION_NOVAMANAGE_CELL_MAPPINGS=$($CONTROLLER1_SSH sudo podman exec -it nova_api nova-manage cell_v2 list_cells)
 echo "$PULL_OPENSTACK_CONFIGURATION_NOVAMANAGE_CELL_MAPPINGS"
 ----
 +
-After the source control plane services shutdown, if either of the exported
-values lost, it could be no longer evaluated again. Preserving the exported
-values in an environment file should protect you from such a situation.
+[IMPORTANT]
+After the {OpenStackShort} control plane services are shut down, if any of the exported
+values are lost, re-running the command fails because the values cannot be evaluated again. To avoid data loss, preserve the exported values in an environment file before shutting down the control plane services.
```
Comment (JoanneOFlynn2018) [inline docs_user/modules/proc_retrieving-topology-specific-service-configuration.adoc:77]: Will the user understand "the values cannot be evaluated again"?

Comment (bogdando) [reply docs_user/modules/proc_retrieving-topology-specific-service-configuration.adoc:77]: I have elaborated the meaning in the comment above: "the values cannot be evaluated again" - if we re-run the commands, they will fail as there are no longer control plane services running on the source cloud to get the data from (and those cannot be started back).

Feel free to rephrase, with preserving the original intent

Comment (klgill) [reply docs_user/modules/proc_retrieving-topology-specific-service-configuration.adoc:77]: Thanks @bogdando . When you say, re-run the commands, do you mean the "export" commands specifically, or any commands in this procedure?

Comment (bogdando) [reply docs_user/modules/proc_retrieving-topology-specific-service-configuration.adoc]: None of the commands in this chapter would work after its relates services stopped on the source cloud

**Pull request merged**